patches/1.6/patch1/arcgis-experience-builder/readme.md

```diff
@@ -0,0 +1,8 @@
+This patch fixes these bugs:
+  * "jimu-icons" is missing error.
+  * Unit test does not work.
+  * Download does not work on Node.js 16 if an app contains custom widgets.
+
+Steps:
+  * Please replace the files in the `client` folder
```

Comment:
We should clarify if you have to run `npm ci` in the client folder again after replacing the files or not.

Reply:
I did have to run `npm ci` in the client after replacing these files for older widgets. Widgets that were first developed against v1.4, otherwise, errors were thrown.

Reply:
Looks strange the `npm ci` is required, but I just update the readme.